setup.cfg

```diff
@@ -1,7 +1,7 @@
 [flake8]
 max-line-length = 110
 max-doc-length = 79
-ignore = E133, E226, E228, N802, N803, N806, N812, N813, N815, N816, W504
+ignore = E133, E226, E228, N802, N803, N806, N812, N813, N815, N816, W504, E401, E303, E302, E305
```

Comment:
Please, remove the additional ignored flake8 tests... without them there's no guarantee the code will follow the project standards...

Reply:
The removed lint tests were too nit-picky, restrictive & provided no added value.